src/devices/src/virtio/net/device.rs

```diff
@@ -34,6 +34,15 @@ use virtio_gen::virtio_net::{
 };
 use vm_memory::{ByteValued, Bytes, GuestAddress, GuestMemoryError, GuestMemoryMmap};
 
+pub enum FrontendError {
+    AddUsed(QueueError),
```

Comment:
The inner `QueueError` is discarded. Any reason to propagate it at all?

Reply:
You're right. I didn't realize that. Fixed.